torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -3349,12 +3357,6 @@ Error PyTorchModelLoader::loadConvolution(const torch::jit::Node *ptNode) {
             getGlowIValueForValue(inputs[ConvInputs::dilation]), 2)));
   }
 
-  // Don't support transposed convolutions yet.
-  bool transposed;
-  ASSIGN_VALUE_OR_RETURN_ERR(transposed, iValToBool(getGlowIValueForValue(
-                                             inputs[ConvInputs::transposed])));
-  RETURN_ERR_IF_NOT(!transposed, "Transposed convolutions not supported.");
-
```

Comment:
Please add if(isConv3d && transpoed) RETURN_ERR here, since we didnt add support for convTranspose3d

Reply:
Good point, added!

Reply:
Good point, added!